<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryFinder</class>
 <widget class="QFrame" name="BinaryFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>126</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="layout">
   <item row="0" column="0">
    <widget class="QLabel" name="findLabel">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="findEdit">
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="findCheckBox">
     <property name="text">
      <string>Ignore Case</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="findButton">
     <property name="text">
      <string>Find</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="replaceLabel">
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="replaceEdit">
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="replaceCheckBox">
     <property name="text">
      <string>Hex Pattern</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="replaceButton">
     <property name="text">
      <string>Replace</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="replaceAllButton">
     <property name="text">
      <string>Replace All</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="directionBox">
     <property name="title">
      <string>Direction</string>
     </property>
     <layout class="QHBoxLayout" name="directionLayout">
      <item>
       <widget class="QRadioButton" name="upButton">
        <property name="text">
         <string>Up</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="downButton">
        <property name="text">
         <string>Down</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QGroupBox" name="originBox">
     <property name="title">
      <string>Origin</string>
     </property>
     <layout class="QHBoxLayout" name="originLayout">
      <item>
       <widget class="QRadioButton" name="beginningButton">
        <property name="text">
         <string>Beginning</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cursorButton">
        <property name="text">
         <string>Cursor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="endButton">
        <property name="text">
         <string>End</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findEdit</tabstop>
  <tabstop>findCheckBox</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>replaceEdit</tabstop>
  <tabstop>replaceCheckBox</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>beginningButton</tabstop>
  <tabstop>cursorButton</tabstop>
  <tabstop>endButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
